phpBB/language/en/acp/common.php

```diff
@@ -289,6 +289,7 @@
 	'PROCEED_TO_ACP'					=> '%sProceed to the ACP%s',
 
 	'REMIND'							=> 'Remind',
+	'REPARSE_LOCK_ERROR'				=> 'Could not obtain reparse lock.',
```

Comment:
This is a very technical message, you should explain what the issue is (ie. an other process might already ben using the reparser)